mockups/search.html

```diff
@@ -1,12 +1,11 @@
 <!DOCTYPE html>
-<html lang="en" dir="ltr">
+<html lang="en" class="no-js" dir="ltr">
```

Comment:
I was trying to add JS and this prevents a page flicker while loading. Probably not necessary in this PR but figured it may be a nice reminder if it's useful in the future.